samza-test/src/main/java/org/apache/samza/system/standalone/ZkTestUtils.java

```diff
@@ -0,0 +1,88 @@
+package org.apache.samza.system.standalone;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.samza.config.ZkConfig;
+
+
+public class ZkTestUtils {
+  private static final String ZK_JOB_COORDINATOR_FACTORY = "org.apache.samza.zk.ZkJobCoordinatorFactory";
```

Comment:
May be there is someother PR that needs to be reviewed before this one because there is no ZkJobCoordinatorFactory defined in samza master